ansible/lpa_telemetry_backup_deploy.yml

```diff
@@ -2,6 +2,16 @@
 - hosts: secor
   vars_files:
     - "{{inventory_dir}}/secrets.yml"
+  pre_tasks:
+    - name: Stop the monit
```

Comment:
@Kaali09 Please add logic to run this block only if monit exists in server. Since adopters may not use monit, this should not be a blocker. But if monit exisits, this block cannot include ignore_errors. If monit fails to start / stop, the deployment should fail. Same for other blocks.

Reply:
Its resolved.